tests/test_metadata_serialization.py

```diff
@@ -254,6 +254,8 @@ def test_invalid_timestamp_serialization(self, test_case_data: str) -> None:
     valid_timestamps: utils.DataSet = {
         "all": '{ "_type": "timestamp", "spec_version": "1.0.0", "version": 1, "expires": "2030-01-01T00:00:00Z", \
             "meta": {"snapshot.json": {"hashes": {"sha256" : "abc"}, "version": 1}}}',
+        "two part spec_version": '{ "_type": "timestamp", "spec_version": "1.9000", "version": 1, "expires": "2030-01-01T00:00:00Z", \
```

Comment:
Nitpick: Maybe better use `1.0` as described in the issue as problematic?
I am not sure there is a point in making spec_version = `1.9000` with so many zeroes helps.

Reply:
I suppose you're right